.github/workflows/python-compatibility.yml

```diff
@@ -0,0 +1,18 @@
+name: 🐍 Python Compatibility Check
+on: [ push ]
+
+jobs:
+  python-compatibility:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ '3.8.10', '3.9', '3.10', '3.11', '3.12', '3.13' ]
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
+
+      - name: Set up Python ${{ matrix.python-version }}
+        run: uv python install ${{ matrix.python-version }}
```

Comment:
todo: add a step to run some form of validation, i.e. run a test script to import the SDK, etc.